model/authentication_http.go

```diff
@@ -102,3 +102,24 @@ func (auth *Authentication) UpdateFromRequest(update *AuthenticationEditRequest)
 
 	return nil
 }
+
+func (auth *Authentication) UpdateSecretFromRequest(update *SecretEditRequest) error {
+	if update.Password != nil {
+		encrypted, err := util.Encrypt(*update.Password)
+		if err != nil {
+			return err
+		}
+		auth.Password = &encrypted
+	}
+
+	if update.Extra != nil {
+		var err error
+		auth.ExtraDb, err = json.Marshal(*update.Extra)
+
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
I am curious if there is some special reason to not have only
`auth.ExtraDb, err := json.Marshal(*update.Extra)`
(because we usually use short variable declaration in this cases)

Reply:
it looks that it is not possible:
`Syntax error: unexpected end of the statement, expecting ':=','=', or ','`

Probably - it is because you need var on left side instead of `auth.ExtraDb`.
(I just removed empty line)

Reply:
ok, thanks